assets/tests/abac_tests.yaml

```diff
@@ -530,7 +530,59 @@ tests:
                 relation: viewer
                 user: user:jon
             expectation: true
-  - name: direct_relations_with_condition
+        listObjectsAssertions:
+          - request:
+              user: user:jon
+              type: document
+              relation: viewer
+            errorCode: 2000
+          - request:
+              user: user:jon
+              type: document
+              relation: viewer
+            context:
+              s: "hello"
+            errorCode: 2000
+          - request:
+              user: user:jon
+              type: document
+              relation: viewer
+            context:
+              s: "not hello"
+            errorCode: 2000
+          - request:
+              user: user:jon
+              type: document
+              relation: viewer
+            context:
+              x: 10
+            errorCode: 2000
```

Comment:
This should be `document:1` not an error

Reply:
Tracking via https://github.com/openfga/openfga/issues/1511